### DISCUSSION TITLE
App refusing to connect to any platform

### DISCUSSION THREAD

**Discussion by FranciscoAmestoy** [3dproger/AxelChat#515] [General]
I checked internet connection, correct username spelling, no need for VPN because I can access all platforms directly, running the app as admin and exception created in firewall. Still, "no connection" error messages, "failed to connect" and eternal "login is in progress"

**Comment by 3dproger** [comment]
Hello!

Please tell me the AxelChat version and could you send a screenshot of the window with logs, as in the image below. There should be errors displayed when trying to connect.
![изображение](https://github.com/user-attachments/assets/15d21e32-11ed-4729-aa1a-a8b04e9afff4)
In version 0.37.3 and later the button for open logs is located here:
![изображение](https://github.com/user-attachments/assets/ba3cf38d-ef34-4483-97f6-c14da8a2854a)
In previous versions it was located here:
![изображение](https://github.com/user-attachments/assets/6e5a2b67-3a74-4379-9d82-98ea16baf9ea)

**Comment by 3dproger** [comment]
Another user has a similar problem: https://github.com/3dproger/AxelChat/issues/519 Probably the same problem. I will try to fix it in one of the future updates

**Comment by 3dproger** [comment]
Apparently, this discussion is no longer relevant, I will close this thread. If you want to write something else, you can reopen the discussion or create a new one
